lib/kasket/read_mixin.rb

```diff
@@ -66,6 +66,8 @@ def find_by_sql_with_kasket(*args)
     def find_by_sql_with_kasket_on_id_array(keys)
       begin
         key_attributes_map = Kasket.cache.read_multi(*keys)
+      rescue Dalli::DalliError, Dalli::MarshalError, Dalli::NetworkError, Dalli::RingError, Dalli::UnmarshalError, Dalli::ValueOverMaxSize
```

Comment:
`Dalli::MarshalError, Dalli::NetworkError, Dalli::RingError, Dalli::UnmarshalError` are all `DalliError` descendants.
I believe `rescue Dalli::DalliError, Dalli::ValueOverMaxSize` should suffice here.

Reply:
Thank you, great point.

Useless trivia: starting in 2.7.11 Dalli has made ValueOverMaxSize inherit from DalliError as well instead of RuntimeError.  I believe leaving Dalli::ValueOverMaxSize in place is still the right thing to do for backward compatibility.

Reply:
Oh.. I haven't realized that. Only checked the current version being used by Classic(I'm lazy). I agree with you.